common/src/main/templates/io/netty/util/collection/KObjectHashMap.template

```diff
@@ -608,7 +606,7 @@ public class @K@ObjectHashMap<V> implements @K@ObjectMap<V> {
             if (nextIndex == -1) {
                 scanNext();
             }
-            return nextIndex < keys.length;
+            return nextIndex != values.length;
```

Comment:
@Scottmitch so we are sure nextIndex can never be > values.length ?  Just want to make sure we are :)